webpages/popup/index.js

```diff
@@ -68,17 +68,16 @@ const vue = new Vue({
 
 let manifests = null;
 // If order unspecified, addon goes first. All new popups should be added here.
```

Comment:
This comment can be changed because the Addons tab is always the first now.
```suggestion
// All new popups should be added here.
```

Reply:
I think the comment is still correct: if an addon isn't included in the TAB_ORDER array, it will become the first tab.

Reply:
This happens if `scratch-messaging` is removed from the array:
![image](https://github.com/user-attachments/assets/2d3649b9-6a9c-478c-824b-e9c551fb8339)

Reply:
Oh, I see.